app/redux/sagas/SyncSaga.js

```diff
@@ -90,6 +97,13 @@ function* pushUpdates() {
 function* pullUpdates(previousRevision=null) {
     var revision = yield select(SetsSelectors.getRevision);    
     const accessToken = yield select(AuthSelectors.getAccessToken);
+    const lastRefreshDate = yield select(AuthSelectors.getLastRefreshDate);
+
+    const validator = {
+        accessToken,
+        lastRefreshDate,
+        isValid: Validator.validator
+    };
```

Comment:
Same deal here, validator shouldn't be an object containing the Validator, it should be the Validator.